frontend/templates/admin/dashboard.html

```diff
@@ -0,0 +1,138 @@
+{% extends "layout.html" %}
+{% block title %}light and sound - admin{% endblock title %}
+{% block styles %}
+  {% call super() %}
+{% endblock styles %}
+{% block content %}
+  <div id="admin/">
+    <aside id="sidebar">
```

Comment:
It would be nice to extend the sidebar to /events and /posts, but that would be dependent on the role I guess? Maybe more trouble than it's worth.